test/integration/users.test.js

```diff
@@ -1441,6 +1447,56 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users?departed", function () {
+    beforeEach(async function () {
+      await cleanDb();
+      const userPromises = abandonedUsersData.map((user) => userModel.doc(user.id).set(user));
+      await Promise.all(userPromises);
+
+      const taskPromises = abandonedTasksData.map((task) => taskModel.add(task));
+      await Promise.all(taskPromises);
+    });
+
+    afterEach(async function () {
+      Sinon.restore();
+      await cleanDb();
```

Comment:
we should add test cases for
- add tests for invalid query parameter cases
- Scenarios where large datasets are involved

Reply:
I don’t see large dataset scenarios implemented in the current tests. Can you point out where it’s being used? 
Also, what scale are we considering for large datasets? Are we talking thousands, tens of thousands or more for mocking data? 
If it’s not covered in existing system, I can add it as an enhancement task for existing api's including this.

Reply:
Have raised this issue here - https://github.com/Real-Dev-Squad/website-backend/issues/2273